vignettes/download_github_comments.Rmd

```diff
@@ -218,9 +222,9 @@ head(all_issues_combined) %>%
 
 The Refresh parser also parses issue data but the folder 'refresh_issues', parsing data retrieved from the search endpoint. In this notebook, that is data from the REFRESH issues portion saved to issue_search directory. 
 
-# Comments
 
-## Issues and Pull Requests Comments
+# Comments
+## Issues and PR Comments by Date Range
```

Comment:
I think format information would be better in a wiki for date-time formats, though discuss that with Carlos- as I am speaking on what he suggested for me to do.